invenio/modules/comments/api.py

```diff
@@ -411,9 +412,9 @@ def check_user_can_vote(cmt_id, client_ip_address, uid=-1):
     cmt_id = wash_url_argument(cmt_id, 'int')
     client_ip_address = wash_url_argument(client_ip_address, 'str')
     uid = wash_url_argument(uid, 'int')
-    query = """SELECT id_cmtRECORDCOMMENT
-               FROM cmtACTIONHISTORY
-               WHERE id_cmtRECORDCOMMENT=%s"""
+    query = """SELECT "id_cmtRECORDCOMMENT"
```

Comment:
Yes, I know. :)
But, we are so close to have a "usable" version with PostgreSQL.
So, I proposed to parallel working on:

 1. Remove raw sql
 2. Make invenio compatible as possible with PostgreSQL

Because to wait of completely remove all raw sql, require to much time and we can instead start playing soon with PostgreSQL.